icons/home-plug.json

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jguddas",
+    "karsa-mistmere"
```

```suggestion
    "jguddas"
```

I think there were definitely some other contributors there (I mostly just optimized stuff in #2116), although I'm finding it hard to decide whether they should even be kept, since you've basically redesigned the entire icon from scratch. 🤔

---

I did not base the redesign on the current home icon.

---

Yeah, that's also my point. :)